cob_light/CMakeLists.txt

```diff
@@ -36,9 +36,32 @@ catkin_package(
 #pkg_check_modules(GTHREAD REQUIRED gthread-2.0)
 
 include_directories(${PROJECT_SOURCE_DIR}/ros/include ${PROJECT_SOURCE_DIR}/common/include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
+set(HEADERS
+    ${PROJECT_SOURCE_DIR}/ros/include/colorO.h
+    ${PROJECT_SOURCE_DIR}/ros/include/colorOSim.h
+    ${PROJECT_SOURCE_DIR}/ros/include/iColorO.h
+    ${PROJECT_SOURCE_DIR}/ros/include/ms35.h
+    ${PROJECT_SOURCE_DIR}/ros/include/stageprofi.h
+    ${PROJECT_SOURCE_DIR}/common/include/breathColorMode.h
+    ${PROJECT_SOURCE_DIR}/common/include/breathMode.h
+    ${PROJECT_SOURCE_DIR}/common/include/circleColorMode.h
+    ${PROJECT_SOURCE_DIR}/common/include/colorUtils.h
+    ${PROJECT_SOURCE_DIR}/common/include/concurrentQueue.h
+    ${PROJECT_SOURCE_DIR}/common/include/fadeColorMode.h
+    ${PROJECT_SOURCE_DIR}/common/include/flashMode.h
+    ${PROJECT_SOURCE_DIR}/common/include/mode.h
+    ${PROJECT_SOURCE_DIR}/common/include/modeExecutor.h
+    ${PROJECT_SOURCE_DIR}/common/include/modeFactory.h
+    ${PROJECT_SOURCE_DIR}/common/include/modeRunner.h
+    ${PROJECT_SOURCE_DIR}/common/include/sequenceMode.h
+    ${PROJECT_SOURCE_DIR}/common/include/serialIO.h
+    ${PROJECT_SOURCE_DIR}/common/include/staticMode.h
+    ${PROJECT_SOURCE_DIR}/common/include/sweepColorMode.h
+)
+
 
 add_executable(cob_light ros/src/cob_light.cpp ros/src/colorO.cpp ros/src/colorOSim.cpp ros/src/ms35.cpp ros/src/stageprofi.cpp
-                         common/src/serialIO.cpp common/src/modeExecutor.cpp common/src/modeFactory.cpp)
+                         common/src/serialIO.cpp common/src/modeExecutor.cpp common/src/modeFactory.cpp ${HEADERS})
```

Comment:
Yes, you have do add the headers to add_executable if you want them to show up in qtcreator.